compiler/luci/pass/src/QuantizationUtils.h

```diff
@@ -47,6 +47,40 @@ bool is_weights(CircleNode *node);
 
 bool is_quantized(const CircleNode *node);
 
+// Clone a new const node from an existing node.
+// The new node has the same shape, type, contents, and qparam with 'node'
+template <loco::DataType T> luci::CircleConst *clone_const_from(luci::CircleConst *node)
+{
+  assert(T == node->dtype());
+
+  auto new_node = node->graph()->nodes()->create<CircleConst>();
+  // TODO: We don't have any naming policy for newly created nodes yet.
+  //       Fix this when we have one.
+  new_node->name(node->name());
+  new_node->dtype(node->dtype());
+  new_node->rank(node->rank());
+  for (uint32_t i = 0; i < node->rank(); i++)
+    new_node->dim(i).set(node->dim(i).value());
+
+  const auto size = node->size<T>();
+  new_node->size<T>(size);
+  for (uint32_t i = 0; i < size; i++)
+    new_node->at<T>(i) = node->at<T>(i);
+
+  new_node->shape_status(luci::ShapeStatus::VALID);
+
+  auto qparam = std::make_unique<CircleQuantParam>();
```

Comment:
maybe do this and below only when `node->quantparam()` is not null?

Reply:
or if `node->quantparam()` be always exist then add `assert(node->quantparam() != nullptr);` at the top would be nice.

Reply:
I've added the null check. Thanks.